port/win/port_win.h

```diff
@@ -241,6 +241,15 @@ extern void InitOnce(OnceType* once, void (*initializer)());
 #define CACHE_LINE_SIZE 64U
 #endif
 
+// To reduce false cache sharing by enforcing alignment
+#if defined(__clang__) || defined(__GNUC__)
+# define CACHE_ALIGNED(size) __attribute__((__aligned__(size)))
+#elif defined(_MSC_VER)
+# define CACHE_ALIGNED(size) __declspec(align(size))
+#else
+# error Cannot define CACHE_ALIGNED on this platform
```

Comment:
Just make it an empty macro? We don't want to make rocksdb uncompilable on other platforms.